### PR TITLE
Moving load level test from sandbox to smoke with fixes

### PR DESCRIPTION
**Pull request by evanchia-ly-sdets**
Signed-off-by: evanchia <evanchia@amazon.com>

Fixed the nullrenderer flag. Added new test that enables the flag with separate CTest registrations. Moved the test from sandbox to smoke suite

### CODE REVIEW

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        request.addfinalizer(teardown)
+
+        return console
+
+    def test_RemoteConsole_LoadLevel_Works(self, launcher, level, remote_console_instance, launcher_platform):
+        expected_lines = ['Level system is loading "Simple"']
+
+        self.launch_and_validate_results_launcher(launcher, level, remote_console_instance, expected_lines)
+
+    def launch_and_validate_results_launcher(
```
Comment (jckand-amzn) [inline AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:47]: Why not use this from editor_python_test_tools.hydra_test_utils? Looks like it was copied out from there and will just double needed maintenance in the future.

Comment (jckand-amzn) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:47]: Looks like the GPU version of the test does the same thing. so would be tripled maintenance.

File: AutomatedTesting/Gem/PythonTests/smoke/CMakeLists.txt [hunk 2]
```diff
@@ -32,20 +32,20 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
...
-        PATH ${CMAKE_CURRENT_LIST_DIR}
-        PYTEST_MARKS "SUITE_sandbox"
-        TIMEOUT 1500
+        TEST_REQUIRES gpu
+        PATH ${CMAKE_CURRENT_LIST_DIR}/test_RemoteConsole_GPULoadLevel_Works.py
+        TIMEOUT 100
         RUNTIME_DEPENDENCIES
             AZ::AssetProcessor
             AZ::PythonBindingsExample
             Legacy::Editor
```
Comment (jckand-amzn) [inline AutomatedTesting/Gem/PythonTests/smoke/CMakeLists.txt:44]: Does this actually need the Editor?

**Review by jckand-amzn — CHANGES_REQUESTED** [review]

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py (hunk 1 above)
Comment (evanchia-ly-sdets) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py]: using import

File: AutomatedTesting/Gem/PythonTests/smoke/CMakeLists.txt (hunk 2 above)
Comment (evanchia-ly-sdets) [reply AutomatedTesting/Gem/PythonTests/smoke/CMakeLists.txt]: removed editor dependency

**Review by jckand-amzn — APPROVED** [review]

File: AutomatedTesting/Gem/PythonTests/smoke/CMakeLists.txt [hunk 3]
```diff
@@ -32,20 +32,19 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
...
-        TEST_SUITE sandbox
+        NAME AutomatedTesting::LoadLevelGPU
+        TEST_SUITE smoke
         TEST_SERIAL
-        PATH ${CMAKE_CURRENT_LIST_DIR}
-        PYTEST_MARKS "SUITE_sandbox"
-        TIMEOUT 1500
+        TEST_REQUIRES gpu
+        PATH ${CMAKE_CURRENT_LIST_DIR}/test_RemoteConsole_GPULoadLevel_Works.py
+        TIMEOUT 100
```
Comment (Kadino) [inline AutomatedTesting/Gem/PythonTests/smoke/CMakeLists.txt:40]: Two minute max timeout seems a little low knowing the inconsistent editor perf ...but I'm happy to have a fairly strict bar for the smoke tests!

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+UI Apps: AutomatedTesting.GameLauncher
+Launch AutomatedTesting.GameLauncher with Simple level
+Test should run in both gpu and non gpu
+"""
+
+import pytest
+import psutil
+
+# Bail on the test if ly_test_tools doesn't exist.
+pytest.importorskip("ly_test_tools")
```
Comment (Kadino) [inline AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:17]: Please delete calls to importorskip wherever you find them. They are mildly dangerous and can lead to users accidentally disabling tests and not knowing it.

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
+"""
```
Comment (Kadino) [inline AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:1]: Why do GPU and CPU variants/duplicates need to exist? Is the CPU variant registered anywhere?

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+                console.stop()
+
+        request.addfinalizer(teardown)
+
+        return console
+
+    def test_RemoteConsole_LoadLevel_Works(self, launcher, level, remote_console_instance, launcher_platform):
+        expected_lines = ['Level system is loading "Simple"']
+
+        editor_test_utils.launch_and_validate_results_launcher(launcher, level, remote_console_instance, expected_lines, null_renderer=True)
```
Comment (Kadino) [inline AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:46]: This is a 45 lines of duplication to change this one null_renderer flag

**Review by Kadino — CHANGES_REQUESTED** [review]

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py (hunk 5 above)
Comment (evanchia-ly-sdets) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:1]: The linked smoke test doc had requirements for both cpu + gpu. The cpu tests should be ran during AR submission

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py (hunk 6 above)
Comment (evanchia-ly-sdets) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:46]: Yeah, but there needs to be a separate file to register it to both cpu/gpu so I'm not sure of any cleaner resolutions

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py (hunk 4 above)
Comment (evanchia-ly-sdets) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:17]: Will remove

**Review by AMZN-scspaldi — APPROVED** [review]

File: AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py (hunk 5 above)
Comment (Kadino) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:1]: Are the CPU tests run during submission? The CMakeLists.txt appears to only reference GPU.

Comment (Kadino) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:1]: GH won't let me comment directly on the line, but there appears to be a different registration block that is registering everything in the directory to run in the Smoke suite, which will probably cause the GPU test to also run in the smoke suite.

Comment (evanchia-ly-sdets) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:1]: The smoke test registration has the "SUITE_smoke" pytest mark requirment. The gpu test does not have that mark and is registered separately so it won't run in the same suite

Comment (evanchia-ly-sdets) [reply AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py:1]: And yes cpu tests are ran during submission. Here is an sample AR run for this PR and you can see there is a cpu profile job https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE/detail/PR-1997/4/pipeline/528

**Review by Kadino — APPROVED** [review]
Looks good. If there's any pipeline-level snags, they will shake out soon enough.

**Pull request merged**